### DISCUSSION TITLE
Does alwaysOnTop setting really mean all windows for all applications?

### DISCUSSION THREAD

**Discussion by david-mkl** [tauri-apps/tauri#9685] [Q&A] [answered]
I wanted to use the `alwaysOnTop`/`always_on_top` option on a new webview window, but it seems like it really does mean _always on top_, i.e. even staying in front when other applications are focused.

I expected this option to mean that the window always stays on top of other windows in the same application. Not every window for every application.

Is what I'm seeing a bug, or is it the intended behavior of this option? (I can't really tell from the documentation) This is on macOS Sonoma and Tauri 2.0.0-beta.17

**Comment by FabianLars** [comment, marked as answer]
> Is what I'm seeing a bug, or is it the intended behavior of this option?

This is intended behavior. Tauri does not have an app-wide z-order mechanism right now (though parent/owner apis may work, not sure 🤔 )

**Reply by david-mkl** [reply]
`parent` is exactly what I wanted. Thank you!

**Reply by alexiokay** [reply]
Works, but I cant drag it anymore
